lib/rufo/dot_file.rb

```diff
@@ -6,9 +6,7 @@ def initialize
   end
 
   def find_in(dir)
-    @cache.fetch(dir) do
-      @cache[dir] = internal_find_in(dir)
-    end
+    @cache[dir] ||= internal_find_in(dir)
```

Comment:
It _stores_ `nil`, but it does not _memoize_, because `nil` is falsey and thus the right hand side of a `nil ||= blah` statement is re-evaluated.